fred2/management.cpp

```diff
@@ -417,6 +417,11 @@ bool fred_init(std::unique_ptr<os::GraphicsOperations>&& graphicsOps)
 	parse_init();
 	techroom_intel_init();
 
+	// get fireball IDs for sexpression usage
+	// (we don't need to init the entire system via fireball_init, we just need the information)
+	extern void fireball_parse_tbl();
```

Comment:
Maybe this function could be added to the fireball header to avoid the extern reference here?